src/EditorFeatures/Core/Shared/Extensions/GlyphExtensions.cs

```diff
@@ -141,12 +141,16 @@ public static ImageId GetImageId(this Glyph glyph)
                     return new ImageId(ImageCatalogGuid, KnownImageIds.Namespace);
 
                 case Glyph.MethodPublic:
+                case Glyph.ConstructorPublic:
                     return new ImageId(ImageCatalogGuid, KnownImageIds.MethodPublic);
```

Comment:
this seems strange.  Is there a reason we map these to methods, instead of types (since consturctor make types)?

Reply:
Because that's the behavior today.